plugins/funind/indfun_common.ml

```diff
@@ -178,46 +178,38 @@ let get_proof_clean do_reduce =
   result
 
 let with_full_print f a =
+  let open Printoptions in
+  let saved_opts = get_current_options () in
   let old_implicit_args = Impargs.is_implicit_args ()
   and old_strict_implicit_args =  Impargs.is_strict_implicit_args ()
   and old_contextual_implicit_args = Impargs.is_contextual_implicit_args () in
-  let old_rawprint = !Flags.raw_print in
-  let old_printuniverses = !Constrextern.print_universes in
-  let old_printallowmatchdefaultclause = !Detyping.print_allow_match_default_clause in
-  Constrextern.print_universes := true;
-  Detyping.print_allow_match_default_clause := false;
-  Flags.raw_print := true;
+  set_printing_all ();
+  let opts = get_current_options () in
+  set_current_options { opts with
+      printing_projections = false;
+      printing_notations = false;
+    };
```

Comment:
This seems wrong, those two options are already `false` with `printing_all`.

Reply:
Don't know why I put those there.